lib/structures/GuildChannel.js

```diff
@@ -42,68 +40,41 @@ class GuildChannel extends Channel {
+     * @arg {Number?} [options.bitrate] [VS] The bitrate of the channel
+     * @arg {Number?} [options.defaultAutoArchiveDuration] [TAFM] The default duration of newly created threads in minutes to automatically archive the thread after inactivity (60, 1440, 4320, 10080)
+     * @arg {Number} [options.defaultForumLayout] [F] The default post [layout type](https://discord.dev/resources/channel#channel-object-forum-layout-types)
+     * @arg {Object?} [options.defaultReactionEmoji] [FM] The [emoji](https://discord.dev/resources/channel#default-reaction-object) to show in the Add Reaction Button
+     * @arg {Number?} [options.defaultSortOrder] [FM] The default post sort type (0 for latest activity, 1 for creation date)
+     * @arg {Number} [options.defaultThreadRateLimitPerUser] [TFM] The default rate limit duration for newly created threads
+     * @arg {Number} [options.flags] [FM/Th] The channel's flag [bit set](https://discord.dev/resources/channel#channel-object-channel-flags)
+     * @arg {Boolean} [options.invitable] [Th] Whether non-moderators can add other non-moderators to the channel
+     * @arg {Boolean} [options.locked] [Th] The lock status of the channel
+     * @arg {String} [options.name] [All/Th] The name of the channel
+     * @arg {Boolean?} [options.nsfw] [TVASFM] The NSFW status of the channel
+     * @arg {Array<PermissionOverwrite | Object>?} [options.permissionOverwrites] [All] An object containing [permission overwrite](https://discord.dev/resources/channel#overwrite-object) objects
+     * @arg {String?} [options.parentID] [TVASFM] The ID of the parent channel category for this channel
+     * @arg {Number?} [options.position] [All] The sorting position of the channel
+     * @arg {Number?} [options.rateLimitPerUser] [TVSFM/Th] The time in seconds a user has to wait before sending another message (does not affect bots or users with manageMessages/manageChannel permissions)
+     * @arg {String?} [options.rtcRegion] [VS] The RTC region ID of the channel (automatic if `null`)
+     * @arg {String?} [options.topic] [TAFM] The topic of the channel
+     * @arg {Boolean} [options.type] [TA] The type of channel (only between types 0 and 5)
+     * @arg {Number?} [options.userLimit] [VS] The channel user limit
+     * @arg {Number?} [options.videoQualityMode] [VS] The camera video quality mode of the channel. `1` is auto, `2` is 720p
      * @arg {String} [reason] The reason to be displayed in audit logs
-     * @returns {Promise<CategoryChannel | GroupChannel | TextChannel | TextVoiceChannel | NewsChannel | NewsThreadChannel | PrivateThreadChannel | PublicThreadChannel>}
+     * @returns {Promise<CategoryChannel | GroupChannel | NewsChannel | NewsThreadChannel | PrivateThreadChannel | PublicThreadChannel | TextChannel | TextVoiceChannel>}
```

Comment:
This still says TextVoiceChannel

Reply:
I was going to go through all of docs anyways since all I did was copy stuff over but thanks for the heads up